control-plane/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+package-lock.json
+firebase-debug.log
+.firebase
+.github
```

Comment:
Please remove this one, we need to be able to track files in `.github`.

Reply:
Done